File: swagger/models/autopay_schedule.py
from .base_model import BaseModel


class AutopaySchedule(BaseModel):
	"""
	no description available

	Attributes:

		frequency_type
		Defines how often clients are charged. Possible values are:  *
		SetNumberOfAutopays  * MonthToMonth

		frequency_value
		The interval of AutoPay frequency, combined with `FrequencyTimeUnit`.
		This value is null if `FrequencyType` is `MonthToMonth`.

		frequency_time_unit
		Defines the time unit that sets how often to run the AutoPay, combined
		with `FrequencyValue`. This value is null if `FrequencyType` is
		`MonthToMonth`. Possible values are:  * Weekly  * Monthly  * Yearly
	"""

	input_map = {
		'FrequencyType': 'frequency_type',
		'FrequencyValue': 'frequency_value',
		'FrequencyTimeUnit': 'frequency_time_unit',
		}

	output_map = {
		'frequency_type': 'FrequencyType',
		'frequency_value': 'FrequencyValue',
		'frequency_time_unit': 'FrequencyTimeUnit',
		}

	frequency_type = None
	frequency_value = None
	frequency_time_unit = None

File: swagger/Models/AutopaySchedule.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class AutopaySchedule extends BaseModel
{
	/**
	 * Defines how often clients are charged. Possible values are:  *
	 * SetNumberOfAutopays  * MonthToMonth
	 * 
	 * @var string
	 */
	public $frequencyType;
	/**
	 * The interval of AutoPay frequency, combined with `FrequencyTimeUnit`.
	 * This value is null if `FrequencyType` is `MonthToMonth`.
	 * 
	 * @var integer
	 */
	public $frequencyValue;
	/**
	 * Defines the time unit that sets how often to run the AutoPay, combined
	 * with `FrequencyValue`. This value is null if `FrequencyType` is
	 * `MonthToMonth`. Possible values are:  * Weekly  * Monthly  * Yearly
	 * 
	 * @var string
	 */
	public $frequencyTimeUnit;

	protected function getInputMap()
	{
		return [
			'FrequencyType' => 'frequencyType',
			'FrequencyValue' => 'frequencyValue',
			'FrequencyTimeUnit' => 'frequencyTimeUnit',
			];
	}

	protected function getOutputMap()
	{
		return [
			'frequencyType' => 'FrequencyType',
			'frequencyValue' => 'FrequencyValue',
			'frequencyTimeUnit' => 'FrequencyTimeUnit',
			];
	}
}
